src/components/ContentWrapper.jsx

```diff
@@ -47,9 +47,10 @@ const ContentWrapper = (props) => {
   const pipelineStatusKey = pipelineStatus.pipeline?.status;
   const pipelineErrors = ['FAILED', 'TIMED_OUT', 'ABORTED'];
 
-  const pipelineRunning = pipelineStatusKey === 'RUNNING';
-  const pipelineRunningError = pipelineErrors.includes(pipelineStatusKey);
-
+  // const pipelineRunning = pipelineStatusKey === 'RUNNING';
+  // const pipelineRunningError = pipelineErrors.includes(pipelineStatusKey);
+  const pipelineRunning = false;
+  const pipelineRunningError = false;
```

Comment:
@cosa65 do we need that merged in?

Reply:
I changed this temporarily so that staging can be opened I intended to revert it before merging.

Reply:
sounds good, just make sure you don't forget about it